test/modelcheck/mps-prj/.mps/markdown-navigator.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files seem unrelated to the PR (I'm fine with adding them but let's maybe do it in a separate commit or, ideally, separate PR?)